app/controllers/coronavirus_form/essential_supplies_controller.rb

```diff
@@ -18,12 +18,12 @@ def submit
       respond_to do |format|
         format.html { render controller_path, status: :unprocessable_entity }
       end
+    elsif session[:check_answers_seen] && session[:dietary_requirements].present? && session[:carry_supplies].present?
```

Comment:
So the user can only go back to `check_answers` directly if they've already answered the `dietary_requirements ` and `carry_supplies` questions? What if they decide not to change their answer to `essential_supplies` and they submit a `yes` again?

Reply:
From the check answers page, they can go directly to the later questions if they want to change those questions, since as far as the user is aware these are individual questions rather than linked in some way.  I've only included this logic so they actually get asked the later questions if they are setting the answer to "Yes" for the first time.

Reply:
I think we've got some crossed wires. If the user previous answered "yes" to `essential_supplies`, completes the form (sees check answers), clicks change their answer to `essential_supplies`, but then answers "yes" again, they should be routed back to check your answers, not `basic care needs`.

Reply:
Got it!  Was getting my yes and no responses mixed up, until I read the question properly 🤦.